docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -93,6 +101,10 @@ The `STATE` parameter mitigates [CSRF attacks](https://en.wikipedia.org/wiki/Cro
 
 1. Click the **Authenticator Config** tab and set the Invoke URL to a URL that _points_ to your application. <mark>Isn't this done when you create the app because the Invoke URL field is required? Could we say "...make sure the Invoke URL points to your application"?</mark>
```

Comment:
Yes. If we add Invoke URL in the "Use case details" section above, then I think this is a good suggestion.